django/contrib/postgres/indexes.py

```diff
@@ -1,8 +1,40 @@
 from __future__ import unicode_literals
 
-from django.db.models import Index
+from django.db.models.indexes import Index
```

Comment:
Not really. They're the same (`__init__.py` has two lines importing this). I can do a Refs-commit, if it's desired?